probe-rs/src/architecture/riscv/communication_interface.rs

```diff
@@ -235,14 +224,27 @@ pub struct RiscvCommunicationInterfaceState {
     /// Number of harts
     num_harts: u32,
 
+    /// Describes, which memory access method should be used for a given access width
     memory_access_info: HashMap<RiscvBusAccess, MemoryAccessMethod>,
 
     /// describes, if the given register can be read / written with an
     /// abstract command
     abstract_cmd_register_info: HashMap<RegisterId, CoreRegisterAbstractCmdSupport>,
 
+    /// First scratch register's state
     s0: ScratchState,
+
+    /// Second scratch register's state
     s1: ScratchState,
+
+    /// Bitfield of enabled harts
+    enabled_harts: u32,
```

Comment:
Why is this a bitfield and not at least a bitvec or rather just a vector? We don't need to save bytes :)

Reply:
No idea, preexisting code.

Reply:
Ah sorry, I did not realize :/